Firmware/variants/1_75mm_MK3-EINSy10a-E3Dv6full.h

```diff
@@ -247,6 +252,7 @@
 #define TMC2130_PWM_CLK   (2 * TMC2130_FCLK / TMC2130_PWM_DIV) // PWM frequency (23.4kHz, 35.1kHz, 46.9kHz, 58.5kHz for 12MHz fclk)
 
 #define TMC2130_TPWMTHRS  0         // TPWMTHRS - Sets the switching speed threshold based on TSTEP from stealthChop to spreadCycle mode
+#define TMC2130_TPWMTHRS_E 403
```

Comment:
Suggest comment:
`// Switch from StealthChop to SpreadCycle at around 900mm/min`